packages/bee-q/src/components/tab-group/_storybook/bq-tab-group.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import { html } from 'lit-html';
+import { TAB_SIZE } from '../../tab/bq-tab.types';
+import mdx from './bq-tab-group.mdx';
+
+export default {
+  title: 'Components/Tab group',
+  component: 'bq-tab-group',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    size: { control: 'select', options: [...TAB_SIZE] },
+    // Event handlers
+    bqChange: { action: 'bqChange' },
+    bqFocus: { action: 'bqFocus' },
+    bqBlur: { action: 'bqBlur' },
+  },
+  args: {
+    size: 'small',
+  },
+};
+
+const Template = (args) => {
+  return html`
+    <bq-tab-group
+      value="5"
+      .size=${args.size}
+      @bqChange=${args.bqChange}
+      @bqFocus=${args.bqFocus}
+      @bqBlur=${args.bqBlur}
+    >
+      <bq-tab tab-id="1">Tab</bq-tab>
+      <bq-tab tab-id="2">Tab</bq-tab>
+      <bq-tab tab-id="3">Long Tab name</bq-tab>
+      <bq-tab tab-id="4" disabled>Tab</bq-tab>
+      <bq-tab tab-id="5" active>Tab</bq-tab>
+      <bq-tab tab-id="6">Tab</bq-tab>
+      <bq-tab tab-id="7">Tab</bq-tab>
+      <bq-tab tab-id="8">Tab</bq-tab>
+    </bq-tab-group>
+  `;
+};
+
+export const Default = Template.bind({});
+
+const IconTemplate = (args) => {
+  return html`
+    <bq-tab-group
+      value="5"
+      .size=${args.size}
+      @bqChange=${args.bqChange}
+      @bqFocus=${args.bqFocus}
+      @bqBlur=${args.bqBlur}
+    >
+      <bq-tab tab-id="1"><bq-icon name="activity" slot="icon"></bq-icon>Tab</bq-tab>
```

Comment:
I think the `activity` icon is no longer available, you should use `pulse` now 🙂 

![CleanShot 2023-04-12 at 20 28 24](https://user-images.githubusercontent.com/328492/231536969-a82c0193-2773-4bb7-9c26-5bb7bfabbc55.png)

Reply:
Done